### ISSUE TITLE
Stopped using WAR sub job JAs

### ISSUE THREAD

**Issue by Sinnersslayer** [EasyFarm/EasyFarm#404]
IDK if it's only me, but I have just noticed, that EF stopped to use ALL /WAR JAs.

**Comment by Sinnersslayer** [comment]
Yes, I have the latest varsion and the profile was working for a long time.
It suddenly stopped.

**Comment by Jinko1** [comment]
I do have the issue that it will sometimes not use an ability even though its on cooldown.

**Comment by bluemages** [comment]
I do have the issue that it will sometimes not use an ability even though in provoke
